modules/migration-running-migration-plan-cam.adoc

```diff
@@ -28,19 +28,15 @@ The {mtc-short} web console must contain the following:
 $ oc adm prune images
 ----
 
-. Log in to the {mtc-short} web console.
-. Select a migration plan.
-. Click *Stage* to copy data from the source cluster to the target cluster without stopping the application.
+. Log in to the {mtc-short} web console and click *Migration plans*.
+. Click the Options menu {kebab} beside a migration plan and select *Stage* to copy data from the source cluster to the target cluster without stopping the application.
```

Comment:
If you change the style on Options in the other module, change twice in this module, too.